src/platforms/stm32/ld/linker.ld.S

```diff
@@ -0,0 +1,211 @@
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2009 Uwe Hermann <uwe@hermann-uwe.de>
+ * Copyright (C) 2013 Frantisek Burian <BuFran@seznam.cz>
+ *
+ * This file is customized for AtomVM.
+ *
+ * Copyright (C) 2023 Winford (Uncle Grumpy) <winford@object.stream>
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
```

Comment:
Do we have an issue distributing this under the Apache 2.0 license?  I am not sure what the implications are of this.  @bettio any thoughts?

Reply:
I would like to avoid (L)GPL only sources, in order to keep licensing simple.

Reply:
Then maybe we really need to consider basing this port on something other than libopencm3 as it is licensed under GPL-3.0, LGPL-3.0 licenses.